modules/howtos/pages/subdocument-operations.adoc

```diff
@@ -206,42 +216,42 @@ These operations are logically similar to the _counter_ operation on an entire d
 
-== Executing multiple operations
+== Executing Multiple Operations
 
-Multiple sub-document operations can be executed at once on the same document, allowing you to retrieve or modify several sub-documents at once.
-When multiple operations are submitted within the context of a single _lookup-in_ or _mutate-in_ command, the server will execute all the operations with the same version of the document.
+Multiple Sub-Document operations can be executed at once on the same document, allowing you to retrieve or modify several Sub-Documents at once.
+When multiple operations are submitted within the context of a single _lookup_in_ or _mutate_in_ command, the server will execute all the operations with the same version of the document.
 
-NOTE: Unlike _batched operations_ which is simply a way of sending multiple individual operations efficiently on the network, multiple subdoc operations are formed into a single command packet, which is then executed atomically on the server.
+NOTE: Unlike _batched operations_ which is simply a way of sending multiple individual operations efficiently on the network, multiple Sub-Document operations are formed into a single command packet, which is then executed atomically on the server.
 You can submit up to 16 operations at a time.
 
-When submitting multiple _mutation_ operations within a single _mutate-in_ command, those operations are considered to be part of a single transaction: if any of the mutation operations fail, the server will logically roll-back any other mutation operations performed within the _mutate-in_, even if those commands would have been successful had another command not failed.
+When submitting multiple _mutation_ operations within a single _mutate_in_ command, those operations are considered to be part of a single transaction: if any of the mutation operations fail, the server will logically roll-back any other mutation operations performed within the _mutate_in_, even if those commands would have been successful had another command not failed.
 
-When submitting multiple _retrieval_ operations within a single _lookup-in_ command, the status of each command does not affect any other command.
+When submitting multiple _retrieval_ operations within a single _lookup_in_ command, the status of each command does not affect any other command.
 This means that it is possible for some retrieval operations to succeed and others to fail.
-While their statuses are independent of each other, you should note that operations submitted within a single _lookup-in_ are all executed against the same _version_ of the document.
+While their statuses are independent of each other, you should note that operations submitted within a single _lookup_in_ are all executed against the same _version_ of the document.
 
-[#subdoc-create-parents]
-== Creating parents
+[#subdoc_create_path]
```

Comment:
Existing links which point to the previous Element ID should be updated to the new one. An easy way to find Element ID usages is to do a "search in folder" from the **modules/**.

Reply:
Can't see anything pointing to this yet, but thanks for the heads up :)

Reply:
I could find one instance of the old Element ID https://github.com/couchbase/docs-sdk-python/blame/release/3.0/modules/howtos/pages/subdocument-operations.adoc#L170

Reply:
fixed in https://github.com/couchbase/docs-sdk-python/commit/86d0b43505209588c2744c8c5dec604cf5b417d2